tests/benchmarks/multicore/idle_gpio/remote/boards/nrf54h20dk_nrf54h20_cpurad_s2ram.overlay

```diff
@@ -4,25 +4,25 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
-   / {
-	aliases {
-		sw1 = &button1;
-	};
-	buttons {
-		compatible = "gpio-keys";
-		button1: button_1 {
-			gpios = <&gpio0 9 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
-			label = "Push button 1";
-			zephyr,code = <INPUT_KEY_1>;
-		};
-	};
-   };
+/ {
+        aliases {
```

Comment:
What happened to this file? Please note that TABs shall be used in the Zephyr related code.

Reply:
Fixed, thanks :)